docs/faq.rst

```diff
@@ -334,6 +334,145 @@ After successfull login the domain part will be striped and the rest used as use
 .. _`591`: https://github.com/Mailu/Mailu/issues/591
 .. _`575`: https://github.com/Mailu/Mailu/issues/575
 
+Spam filtering
+--------------
+
+How does spam filtering work in Mailu?
+``````````````````````````````````````
+
+Mailu uses Rspamd for spam filtering. Rspamd is a Fast, free and open-source spam filtering system.
+
+Rspamd rejects non-compliant email messages and email messages that contain virusses. In Mailu Rspamd uses a scoring scale from 0 to 15:
+
+* Email messages with a score of 15 or higher will be rejected.
+
+* Email messages with a score of 6 or higher will receive a header with the spam score.
+
+* Email messages with a score of 4 or higher will be greylisted.
+
+* Email messages with a score of 3 or lower will be accepted.
+
+1. When an email message is received or send by Postfix, it is scanned by Rspamd. If the email message receives a spam score between 6 and 15, a header is added to the email message with the spam score. This is an example of an email header for a spam score of 14:
+
+  ``X-Spam-Level: **************``
```

Comment:
So when I check my mails, the headers look like this:

```
X-Spamd-Bar: ++++++
X-Spam-Level: ******
Authentication-Results: lisa.bayz.de;
	dkim=pass header.d=example.com header.s=mailing header.b=ABCDE;
	dkim=pass header.d=example.com header.s=mailing header.b=ABCDE;
	dmarc=pass (policy=none) header.from=eventim.de;
	spf=pass (lisa.bayz.de: domain of return@example.com designates 11.22.33.44 as permitted sender) smtp.mailfrom=return@example.com
X-Spam: Yes
```

How about you include them as well in your FAQ?

Reply:
That is a good idea. I did not originally  include the full header because I didn't want to bloat the FAQ entry to much. That is not a problem anymore since I'm moving everything to a new page.